package.json

```diff
@@ -9,7 +9,7 @@
     "src/*"
   ],
   "scripts": {
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
Yeah, the tests were interfering with each other otherwise. For example I use `const request = mock.history.post[0];` to get the latest request made and it only works if one test runs at a time. There might be a way around this now that I think about it, though. I'll work on this.

Reply:
That sounds like you leak the mocks somewhere. It shouldn't be an issue and can result in false-positives or false-negatives.

Reply:
No, the problem is that the history gets scrambled with multiple requests running simultaneously, and then the test doesn't know which one to examine. Having one test run at a time and then erasing the history works fine. Open to suggestions though.